### PR TITLE
Closes #1785 Added Node version to readme and added nvmrc file

### LINKED ISSUE #1785: [DOCS] Not Obvious what version of node the project runs on
### Description

Project fails to build if node is v18 
Redeme should be updated with node version

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by greenchapeljohn**
Closes #1785

## Changes proposed

Added to docs to describe the node version and how to install with nvm
Adde nvmrc file

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1786"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ Check out the website: https://linkfree.eddiehub.io
 Before contributing or adding a new feature, please make sure you have already installed the following tools:
 
 - [Git](https://git-scm.com/downloads)
-- [NodeJs](https://nodejs.org/en/download/)
+- [NodeJs](https://nodejs.org/en/download/) (Works with node - lts version v16.17.0)
+- Optional [NVM](https://github.com/nvm-sh/nvm): Switch node version - `nvm use` ( on windows `nvm use v16.17.0`), if this is not installed run `nvm install v16.17.0`.
```
Comment (Panquesito7) [inline README.md:38]: Just a few minor improvements. 🙂
```suggestion
- [NodeJs](https://nodejs.org/en/download/) (Works with Node LTS version v16.17.0)
- Optional [NVM](https://github.com/nvm-sh/nvm): Switch Node version by using `nvm use` (on Windows, use `nvm use v16.17.0`). If this is not installed, run `nvm install v16.17.0`.
```

Comment (greenchapeljohn) [reply README.md]: Thanks, Yes the English is better in this version

**Review by Panquesito7 — APPROVED** [review]
Looks great. Thank you! 🚀

**Review by EmmaDawsonDev — APPROVED** [review]
Looks good 👍

**Pull request merged**